cmd/kube-scheduler/app/server.go

```diff
@@ -111,6 +111,7 @@ for more information about scheduling and the kube-scheduler component.`,
 	cliflag.SetUsageAndHelpFunc(cmd, *nfs, cols)
 
 	if err := cmd.MarkFlagFilename("config", "yaml", "yml", "json"); err != nil {
+		// nolint:logcheck // TODO (?): return error instead of logging it here
```

Comment:
Moving this discussion here: https://github.com/kubernetes/kubernetes/pull/111155#discussion_r1095985798

Reply:
let's use klog.Background